data/tribes/initialization/empire/units.lua

```diff
@@ -1227,13 +1278,21 @@ wl.Descriptions():new_tribe {
       {
          name = "empire_stonemasons_house",
          helptexts = {
+            -- TRANSLATORS: Lore helptext for an empire production site: Stonemason's House
+            lore = pgettext("empire_building", "‘marmoris insania!’ (Marble Madness!)"),
+            -- TRANSLATORS: Lore author helptext for an empire production site: Stonemason's House
+            lore_author = pgettext("A Barabarian about them imperial marbel clumns"),
```

Comment:
@klaus-halfmann Why would a Barbarian speak in Latin?

Reply:
Well it was propably retranslated :-)